samples/Tests/FactSet.MarkdownAndDateTime.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "type": "AdaptiveCard",
+    "version": "1.1",
```

Comment:
Let's include a non-english "lang" in this card to test that language is being propagated correctly to the fact set